backend/src/test/java/de/neuefische/backend/service/CovidTestServiceTest.java

```diff
@@ -0,0 +1,87 @@
+package de.neuefische.backend.service;
+
+import de.neuefische.backend.dto.CovidTestDto;
+import de.neuefische.backend.model.CovidTest;
+import de.neuefische.backend.model.TestType;
+import de.neuefische.backend.repo.CovidTestRepo;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.*;
+
+class CovidTestServiceTest {
+
+    private final CovidTestRepo covidTestRepo = mock(CovidTestRepo.class);
+    private final CovidTestService covidTestService = new CovidTestService(covidTestRepo);
+
+    @Test
+    void getCovidTestsTest() {
+        // GIVEN
+        List<CovidTest> expectedCovidTests = List.of(CovidTest.builder()
+                .testType(TestType.ANTIGEN_NASAL)
+                .dateTime(LocalDateTime.of(2021, 11, 15, 7, 5))
+                .build());
+        when(covidTestRepo.findAll()).thenReturn(expectedCovidTests);
+
+        // WHEN
+        List<CovidTest> actualCovidTests = covidTestService.getCovidTests();
+
+        // THEN
+        assertEquals(expectedCovidTests, actualCovidTests);
+        verify(covidTestRepo).findAll();
```

Comment:
You don't need to verify that call because you already have the `when(covidTestRepo.findAll()).thenReturn(expectedCovidTests);`. So your test would fail anyway, if that is not called in your implementation.